CMakeLists.txt

```diff
@@ -86,8 +86,10 @@ include(cmake/thirdparty/get_nvtx.cmake)
 add_library(rmm INTERFACE)
 add_library(rmm::rmm ALIAS rmm)
 
-target_include_directories(rmm INTERFACE "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-                                         "$<INSTALL_INTERFACE:include>")
+target_include_directories(
+  rmm
+  INTERFACE "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+            "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>" "$<INSTALL_INTERFACE:include>")
```

Comment:
Why did this change?

Reply:
The logger macros like `RMM_LOG_TRACE` are generated via [this CMake function](https://github.com/rapidsai/rapids-logger/blob/main/cmake/create_logger_macros.cmake) and placed into the build directory. Therefore, when building against rmm's build directory, we need to include `${BUILD_DIR}/include`. That was previously being done in the `rmm_logger` target by the old rapids-logger code, but that target no longer exists.